lib/srv/termmanager.go

```diff
@@ -70,17 +71,9 @@ func NewTermManager() *TermManager {
 
 func (g *TermManager) writeToClients(p []byte) int {
 	g.lastWasBroadcast = false
-	truncateFront := func(slice []byte, max int) []byte {
-		if len(slice) > max {
-			return slice[len(slice)-max:]
-		}
-
-		return slice
-	}
-
-	g.history = append(g.history, truncateFront(p, maxHistory)...)
-	if len(g.history) > maxHistory {
-		g.history = g.history[:maxHistory]
+	g.history = append(g.history, truncateFront(p, maxHistoryBytes)...)
+	if len(g.history) > maxHistoryBytes {
+		g.history = g.history[:maxHistoryBytes]
```

Comment:
If I'm reading this right we append to the history, and then truncate it from the end? So once we reach `maxHistoryBytes` we would always append and then immediately truncate off what was just appended, always saving the oldest data, is that right?